linux_os/guide/services/ssh/ssh_server/sshd_limit_user_access/rule.yml

```diff
@@ -19,10 +19,11 @@ severity: unknown
 identifiers:
     cce@rhel6: 27556-0
     cce@rhel7: 80219-9
+    cce@rhel8: 82422-7
 
 references:
     cui: 3.1.12
-    nist: AC-3
+    nist: AC-3,CM-6(a)
```

Comment:
AC-17(a) needs to be added.

Reply:
Earlier review had only ``AC-3,CM-6(a)``